### PR TITLE
availability table component

### PR DESCRIPTION
**Pull request by fadeomar**
***Availability Table Component*** let the users as a therapy to enter the dates that he will not be busy on it via day . :+1: 

***Availability Table Component*** ****contain :****  
  - table contain the days of week 
  - every day has two time picker one for start "from" and another for end  "to"

***Availability Table Component*** ****will look like : :****   
Availability Table Component

** to see and test the component follow the steps :**
  - go to app folder and open the app file 
  - import the availability component to the app 
```import TabelAvailability from '../common/availabilityTable';```
  - put the component inside the app div 
```  
    <div className="App">
        <TabelAvailability />
      </div>
```
  - run the server 
```npm run start```

### CODE REVIEW

File: client/src/components/common/availabilityTable/index.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import 'antd/dist/antd.css';
```
Comment (AbdallahAmmar96) [inline client/src/components/common/availabilityTable/index.js]: put this line in app.js component

File: client/src/components/common/availabilityTable/index.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+import './style.css';
+import { Table, TimePicker } from 'antd';
+import moment from 'moment';
```
Comment (AbdallahAmmar96) [inline client/src/components/common/availabilityTable/index.js]: arrange your import

File: client/src/components/common/availabilityTable/style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+.ant-table-pagination {
+    display: none;
+}
+
+tbody > tr:last-child { 
+    border:none;
+}
+
+
```
Comment (AbdallahAmmar96) [inline client/src/components/common/availabilityTable/style.css]: ?

File: client/src/components/common/availabilityTable/style.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  width: 33%;
+  display: flex;
+  color: rgb(8, 255, 210);
+
+}
+.ant-table-thead > tr:first-child > th:last-child {
+    border-top-right-radius: 20px;
+}
+
+
```
Comment (AbdallahAmmar96) [inline client/src/components/common/availabilityTable/style.css]: ?

File: client/src/components/common/availabilityTable/style.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+.ant-table-thead > tr > th {
+    background: #fff;
+}
+
+th {
+  width: 33%;
+  display: flex;
+  color: rgb(8, 255, 210);
+
```
Comment (MohammadAlhalaq) [inline client/src/components/common/availabilityTable/style.css]: Empty line

File: client/src/components/common/availabilityTable/style.css [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    border-radius:20px; 
+    width: fit-content;
+    border: 0.5px solid gray;
+}
+
+.table__day {
+    color: rgb(30, 31, 31);
+}
+
+
```
Comment (MohammadAlhalaq) [inline client/src/components/common/availabilityTable/style.css]: Empty line

File: client/src/components/App/index.js [hunk 7]
```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-
 import './style.css';
```
Comment (hashemt) [inline client/src/components/App/index.js]: add line before import style

**Pull request merged**